native_client/javascript/client.ts

```diff
@@ -71,6 +72,14 @@ if (args['scorer']) {
   }
 }
 
+if (args['hot_words']) {
+	console.error('Adding hot-words %s', args['hot_words']);
+	for (let word_boost of args['hot_words'].split(',')) {
+		let word = word_boost.split(':');
+		model.addHotWord(word[0], parseFloat(word[1]));
```

Comment:
You actually caught a bug here, in @JRMeyer exposure to JS :)

The `index.ts` definition for `addHotWord` (and others) calls the wrong functions.

Reply:
You need to fix the following call sites:
 - https://github.com/mozilla/DeepSpeech/blob/92e6daeda4196a3f99ed8414f4155787c08bf8e0/native_client/javascript/index.ts#L194
 - https://github.com/mozilla/DeepSpeech/blob/92e6daeda4196a3f99ed8414f4155787c08bf8e0/native_client/javascript/index.ts#L208
 - https://github.com/mozilla/DeepSpeech/blob/92e6daeda4196a3f99ed8414f4155787c08bf8e0/native_client/javascript/index.ts#L220

In each case, the function's first letter is not capitalized when it should be, so you need to change:
 - `binding.addHotWord` to `binding.AddHotWord`
 - `binding.eraseHotWord` to `binding.EraseHotWord`
 - `binding.clearHotWords` to `binding.ClearHotWords`